superbench/config/default.yaml

```diff
@@ -46,9 +46,10 @@ superbench:
     nccl-bw:default:
```

Comment:
Why do we change this?

Reply:
In `test_runner.py`, `test_run_default_benchmarks` will test to run all benchmark in `default.yaml`. This changes is to cover pattern part in mpi mode, thus can increase the coverage in `runner.py`.

Reply:
Got it, thanks.